events/on_member_role_removed_EVT.js

```diff
@@ -9,12 +9,13 @@ module.exports = {
     DBM.Events = DBM.Events || {}
     const { Bot, Actions } = DBM
     DBM.Events.roleRemoved = async function (oldMember, newMember) {
-      const oldRoles = oldMember.roles
-      const newRoles = newMember.roles
+      if (newMember.roles.cache.size > oldMember.roles.cache.size) return
+      const oldRoles = oldMember.roles.cache
+      const newRoles = newMember.roles.cache
 
       const difference = oldRoles.filter((role) => !newRoles.has(role.id)).first()
-      if (newMember.roles.size > oldMember.roles.size) return

```

Comment:
I believe this should stay?

Reply:
Line 12 In the updated document, I moved it up and made it grab `cache`'s size.